lib/temporal.rb

```diff
@@ -36,6 +36,18 @@ def start_workflow(workflow, *input, **args)
       response.run_id
     end
 
+    def terminate_workflow(workflow, workflow_id:, **options)
+      execution_options = ExecutionOptions.new(workflow, options)
+
+      client.terminate_workflow_execution(
+        namespace: execution_options.namespace,
+        workflow_id: workflow_id,
+        run_id: options[:run_id],
```

Comment:
Not sure if `run_id` is actually optional here and what would happen if there are multiple executions?

Reply:
`run_id` is optional: https://github.com/temporalio/sdk-go/blob/27e8be2e856181ef2b97e34690f6ab085d3170ce/client/client.go#L162